inventory/prod/group_vars/credentials.yml

```diff
@@ -0,0 +1,5 @@
+credentials_env:
+  AWS_ACCESS_KEY_ID: "{{ lookup('env', 'AWS_ACCESS_KEY_ID') }}"
+  AWS_SECRET_ACCESS_KEY: "{{ lookup('env', 'AWS_SECRET_ACCESS_KEY') }}"
+  INFLUX_PASSWORD: "{{ influxdb_user_password }}"
```

Comment:
is this variable could be find without lookup `influxdb_user_password`?

Reply:
It is provided by AWX